packages/cli/src/cli/overview/action.create.overview.ts

```diff
@@ -126,8 +126,7 @@ export class CommandCreateOverview extends CommandLineAction {
     }
   }
 
-  async getTileMatrix(sources: ChunkSource[]): Promise<TileMatrixSet> {
-    const tiff = await CogTiff.create(sources[0]);
+  async getTileMatrix(tiff: CogTiff): Promise<TileMatrixSet> {
```

Comment:
this doesnt need to be async the tiff is `tiff.init(true)` before.

(you are initing the tiff twice)

Reply:
I need this to run `await tiff.getImage(0).loadGeoTiffTags();`, before getting the espg. The just tiff.init now working. 
![image](https://user-images.githubusercontent.com/12163920/204701221-94c12d6d-e3a0-4638-9bbf-8ad6aeed0f4d.png)

Reply:
The tiff.init(true) is called in the other function you should pull that out to the same place as the new CogTiff